dashboard/app/models/census/census_summary.rb

```diff
@@ -26,6 +26,67 @@ class Census::CensusSummary < ApplicationRecord
     HISTORICAL_YES: "HY",
     HISTORICAL_NO: "HN",
     HISTORICAL_MAYBE: "HM",
+    EXCLUDED: "E",
+    UNKNOWN: "U",
   }.freeze
   enum teaches_cs: TEACHES
+
+  # Use the zero byte as the quote character to allow importing double quotes
+  #   via http://stackoverflow.com/questions/8073920/importing-csv-quoting-error-is-driving-me-nuts
+  CSV_IMPORT_OPTIONS = {col_sep: ",", headers: true, encoding: 'bom|utf-8'}.freeze
+
+  # Seeds all the data from the S3 file.
+  def self.seed_all
+    Census::CensusSummary.transaction do
+      CDO.log.info "Seeding census summary data."
+      AWS::S3.seed_from_file('cdo-census', "access-report-data-files-2023/final_csv/final_csv_2023_09_19.csv") do |filename|
```

Comment:
I would ask @bakerfranke for a review here as well

Reply:
Yes that would be fantastic in a little bit! But I think GitHub auto-requested review of this PR and it's not quite ready yet since I'm working on seeding some missing schools before proceeding with this PR as it may change some of the context up top

Reply:
having a hardcoded filename with a date in it seems ok to me for now, assuming we are only going to update this once or twice a year. if we are getting more updates than this, let's look into how to make this fully self-serve at that time.